<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackerReportOptions</class>
 <widget class="QDialog" name="TrackerReportOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Tracker Report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QCheckBox" name="m_checkBoxDisplayTracker">
         <property name="text">
          <string>Display Report when complete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_checkBoxInternalRptTracker">
         <property name="text">
          <string>Generate Internal Report</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="m_checkBoxExcelRptTracker">
         <property name="text">
          <string>Generate Excel Report</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Include Status</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QCheckBox" name="m_checkBoxNewTracker">
          <property name="text">
           <string>New</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkBoxAssignedTracker">
          <property name="text">
           <string>Assigned</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkBoxResolvedTracker">
          <property name="text">
           <string>Resolved</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkBoxDeferedTracker">
          <property name="text">
           <string>Defered</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkBoxCancelledTracker">
          <property name="text">
           <string>Cancelled</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Include Types</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QCheckBox" name="m_checkBoxItemsTracker">
          <property name="text">
           <string>Tracker Items</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="m_checkBoxActionTracker">
          <property name="text">
           <string>Action Items</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Email Options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>4</number>
        </property>
        <property name="leftMargin">
         <number>4</number>
        </property>
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="rightMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
        <item>
         <widget class="QRadioButton" name="m_radioBoxEmailAsHTML">
          <property name="text">
           <string>Email as HTML</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_radioBoxEmailAsPDF">
          <property name="text">
           <string>Email as a PDF attachment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_radioBoxEmailAsExcel">
          <property name="text">
           <string>Email as an Excel attachment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_radioBoxDoNotEmail">
          <property name="text">
           <string>Do not email</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
